radicle-cli/src/commands/track.rs

```diff
@@ -0,0 +1,94 @@
+use std::ffi::OsString;
+
+use anyhow::anyhow;
+
+use radicle::identity::project::Id;
+use radicle::node::Handle;
+
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+
+pub const HELP: Help = Help {
+    name: "track",
+    description: "Manage radicle project tracking relationships",
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+
+    rad track <id> [--fetch]
```

Comment:
Current implementation needs `<id>` passed to it, previously it'd have worked in working dir, and it'd have found out the `(urn, repo)` tuple from `project::cwd()`.

I can't find anything like that in `radicle/src/identity/project/id.rs` or `radicle/src/identity/project.rs`.

What should I do?

Reply:
Check `radicle::rad::remote` function